src/NHibernate.Validator/Properties/DefaultValidatorMessages.de.resx

```diff
@@ -135,6 +135,9 @@
   <data name="validator.email" xml:space="preserve">
     <value>not a well-formed email address</value>
   </data>
+  <data name="validator.enum" xml:space="preserve">
+    <value>muss overeenkomen met Enum definition</value>

```

Comment:
met? mit? overeenkomen???

Das ist kein Deutsch.

Better not contribute the translation than contributing bad translations I think. The English version will be shown in such case.

Reply:
I has asked Deutsch speaking friend
He proposed 2 variants
1) "The value must correspond to the difinition of enumeration" --> "Der Wert muss der Enumerationsdefinition entsprechen"
2) "The value must be one of the values from defined list" --> "Der Wert muss einer der Werte aus der definierten Liste sein"

Reply:
Great, 1. seems the best to me, especially if this message is also used for flags.